saleor/checkout/actions.py

```diff
@@ -217,6 +218,63 @@ def transaction_amounts_for_checkout_updated(
     previous_charge_status = checkout_info.checkout.charge_status
     previous_authorize_status = checkout_info.checkout.authorize_status
     fetch_checkout_data(checkout_info, manager, lines, force_status_update=True)
+    _transaction_amounts_for_checkout_updated(
+        transaction,
+        previous_charge_status,
+        previous_authorize_status,
+        checkout_info,
+        lines,
+        manager,
+        user,
+        app,
+    )
+
+
+def transaction_amounts_for_checkout_updated_without_sync_webhooks(
```

Comment:
`_without_sync_webhooks(`
It is true only in case when you're using webhooks. IMHO it is not correct name. We skip the whole potential recalculation, not webhooks.